src/tests/JIT/Regression/JitBlue/GitHub_57065/Runtime_57065.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+// Generated by Fuzzlyn v1.2 on 2021-08-06 19:31:41
+// Run on .NET 6.0.0-dev on Arm Linux
+// Seed: 5500224797583134883
+// Reduced from 62.7 KiB to 0.5 KiB in 00:01:25
+// Debug: Prints 0 line(s)
+// Release: Prints 1 line(s)
+
+struct S0
+{
+    public int F0;
+    public short F1;
+    public int F2;
+    public bool F3;
+}
+
+class C0
+{
+    public ushort F1;
+}
+
+public class Program
+{
+    static S0[, ] s_1 = new S0[, ]{{new S0()}};
+    static C0 s_4 = new C0();
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static int Test()
+    {
+        S0 vr0 = default(S0);
+        long vr1 = vr0.F0++;
+        s_1[0, 0] = vr0;
+        if (s_1[0, 0].F3)
```

Comment:
Field F3 looks to me like it will always be false.

Reply:
Issue is with the release build the condition satisfies and console writeline expression runs (1 line printed in release). Debug build is fine as no line is printed.